save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityGeneralInfoProps.kt

```diff
@@ -348,8 +348,8 @@ val vulnerabilityGeneralInfoProps: FC<VulnerabilityGeneralInfoProps> = FC { prop
                         className = ClassName("font-weight-bold text-primary-blue mb-4")
                         +"Credits"
                     }
-                    cosv.credits!!.forEachIndexed { creditInd, credit ->
-                        val userPrefix = if (credit.name in saveContributors.map { it.name }) "[SAVE] user" else "External user"
+                    cosv.credits!!.distinctBy { it.name }.forEachIndexed { creditInd, credit ->
```

Comment:
I'm not sure that we should hide some content from origin file.
can we merge them at least? if we have a duplicate by `it.name` then merge content of contacts

Reply:
Good point, changed